shenyu-plugin/shenyu-plugin-tars/src/main/java/org/apache/shenyu/plugin/tars/TarsPlugin.java

```diff
@@ -79,9 +97,19 @@ protected Mono<Void> doExecute(final ServerWebExchange exchange, final ShenyuPlu
         Object prx = tarsInvokePrxList.getTarsInvokePrxList().get(index).getInvokePrx();
         Method method = tarsInvokePrxList.getMethod();
         CompletableFuture future;
+
+        initThreadPool();
         try {
-            future = (CompletableFuture) method
-                    .invoke(prx, PrxInfoUtil.getParamArray(tarsInvokePrxList.getParamTypes(), tarsInvokePrxList.getParamNames(), body));
+            future = CompletableFuture.supplyAsync(() -> {
```

Comment:
We should find the implementation of the `proxy`, and then try to use the shared thread pool in the `proxy`, instead of using `CompletableFuture#supplyAsync` directly.